frontend/src/pages/modelRegistry/ModelRegistryCoreLoader.tsx

```diff
@@ -53,16 +57,37 @@ const ModelRegistryCoreLoader: React.FC<ModelRegistryCoreLoaderProps> =
       renderStateProps = {
         empty: true,
         emptyStatePage: (
-          // TODO: Replace this with a component for empty registries once we have the designs
           <EmptyModelRegistryState
-            title="No model registries found"
-            description="No model registries found in the cluster. Configure a new one before registering models."
-            primaryActionText="Configure model registry"
-            primaryActionOnClick={() => {
-              // TODO: Add primary action
-            }}
+            testid="empty-registered-models"
```

Comment:
This is not the correct test-id. We need new Cypress tests with new test-id, for example, `empty-model-registry` or `empty-state-model-registry`(same as the image name).

Reply:
Updated.